src/services/categories.ts

```diff
@@ -21,6 +21,12 @@ const list = (
   return get("categories", query);
 };
 
+const create = (name: string) => {
+  const body = { name };
+  return jsonPost("categories", body);
+};
```

Comment:
You should've tested things locally, this is not working  because you are calling the endpoint wrong

```suggestion
const create = (name: string) => {
  return jsonPost(`categories/${name}`, body);
};
```